scripts/zkai_hybrid_proof_pressure_selector_gate.py

```
@@ -0,0 +1,736 @@
+        raise HybridSelectorError(f"{route_id} proof-size comparability overclaim")
+    if object_class == "local_external_gkr_fixture" and (row["matched_workload"] or row["native_equivalent"]):
+        raise HybridSelectorError(f"{route_id} GKR fixture promoted to matched native route")
+    if "statement" in object_class and row["ratio_vs_stwo_frontier"] != "NA":
+        raise HybridSelectorError(f"{route_id} statement artifact promoted to byte ratio")
+    if row["proof_system"] == "NANOZK" and row["matched_workload"]:
+        raise HybridSelectorError(f"{route_id} NANOZK paper context promoted to matched workload")
+    if row["proof_system"] == "NANOZK" and "not locally reproduced" not in row["non_claims"]:
+        raise HybridSelectorError(f"{route_id} NANOZK local reproduction non-claim missing")
+
+
+def validate_payload(payload: dict[str, Any], *, final: bool = True) -> None:
```

Ruff (0.15.12)

[warning] Too many branches (24 > 12)

(PLR0912)